README.md

```diff
@@ -52,6 +52,11 @@ rm -rf ffmpeg-git-*
 
 # install rust if not installed (ensure you've restarted your terminal after installation)
 curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+```
+
+### Project dependencies installation
```

Comment:
add a small comment here that we provide two options...

Reply:
Added, you can [preview here](https://github.com/metavoiceio/metavoice-src/blob/02c9ba891b77bfa46716febd02d6ab29867a908e/README.md#using-pipconda)